lib/simp/cli/config/items/action/warn_client_yum_config_action.rb

```diff
@@ -10,26 +9,26 @@ def initialize(puppet_env_info = DEFAULT_PUPPET_ENV_INFO)
       @key             = 'yum::repositories::client::config::check'
       @description     = 'Check YUM configuration for SIMP clients'
       @category        = :sanity_check
-      @warning_message = <<DOC
-Unable to verify YUM configuration for SIMP clients.  Please
-manually verify prior to kickstarting SIMP clients.
+      @warning_message = <<~DOC
+        Unable to verify YUM configuration for SIMP clients.
+        Please manually verify prior to kickstarting SIMP clients.
 
-See https://docs.puppet.com/puppet/latest/types/yumrepo.html for
-a description of the Puppet native type for YUM repositories.
-DOC
+        See https://docs.puppet.com/puppet/latest/types/yumrepo.html for
+        a description of the Puppet native type for YUM repositories.
+      DOC
 
       @warning_message_brief = 'Your SIMP client YUM configuration requires manual verification'
    end
 
     def apply
       @applied_status = :deferred
-      warn( "\nWARNING: #{@warning_message.strip}", [:YELLOW] )
+      warn( "\nIMPORTANT: #{@warning_message}".yellow )
       pause(:warn)
     end
 
     def apply_summary
       if @applied_status == :deferred
-         extra = ":\n\t#{@warning_message_brief}"
+         extra = ":\n    #{@warning_message_brief}"
```

Comment:
Shouldn't this be a `\t` instead of a physical tab?

Reply:
It should be 4 spaces, not a physical tab.